km_api/know_me/models.py

```diff
@@ -406,6 +406,26 @@ def get_gallery_url(self, request=None):
             kwargs={'pk': self.pk},
             request=request)
 
+    def get_media_resource_category_url(self, request=None):
```

Comment:
I believe the convention we have would cause this to be named `get_media_resource_category_list_url`.